cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -317,5 +317,17 @@ func writeCSVs(resultPath string, assessments report.AssessmentPerModelPerLangua
 		return pkgerrors.Wrap(err, "could not write models-summed.csv summary")
 	}
 
+	// Write the individual "language-summed.csv" containing the summary per model per language.
+	byLanguage := assessments.CollapseByLanguage()
+	for language, modelsByLanguage := range byLanguage {
+		csvByLanguage, err := report.GenerateCSV(modelsByLanguage)
```

Comment:
Is this commit missing something? I cannot find the header + row implementation of "by language"?

Reply:
It doesn't need it as the language index is only for the different files. The assessment underneath is `AssessmentPerModel` and that has the formatting.